### PR TITLE
Merge intern examples

### PR DESCRIPTION
**Pull request by JamesH65**
I have updated a couple of files per @kilograham requests, but I have not tested any of these changes. They look sane. So from my point of view I think this branch can be merged.

### CODE REVIEW

File: uart/lcd_uart/README.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+= Attaching a 16x2 LCD via TTL
+
+This example code shows how to interface the Raspberry Pi Pico to one of the very common 16x2 LCD character displays. Due to the large number of pins these displays use, they are commonly used with extra drivers or backpacks. In this example, we will use an Adafruit LCD display backpack, which supports communication over USB or TTL. A monochrome display with an RGB backlight is also used, but the backpack is compatible with monochrome backlight displays too. There is <<_attaching_a_16x2_lcd_via_i2c, another example>> that uses I2C to control a 16x2 display.
```
Comment (lurch) [inline uart/lcd_uart/README.adoc:3]: @JamesH65 There's currently no way for a pico-examples `README.adoc` to link to somewhere else in the databook, which means that this link won't work. So `<<_attaching_a_16x2_lcd_via_i2c, another example>>` should be changed to just `another example` .

**Comment by aallan** [comment]
Merged an additional example in that @lurch fixed! :)

**Comment by aallan** [comment]
> Nope, I don't have write-permissions to this repo, so can't push to any branches.

So if we can get  #154, #151, #150, #147, #143 fixed up and merged into the `merge-intern-examples` branch before accepting this PR that'd be great. @liamfraser, @kilograham ..? 

>  And I also don't know how you were able to push a commit to e.g. #150 as that appears to be a branch coming from an external repo? 

I edited in the GitHub UI and it let me? I guess it stored the change somewhere in the PR rather than the external branch, I was sort of surprised as well!

File: i2c/pa1010d_i2c/pa1010d_i2c.c [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        }
+        i++;
+    } 
+}
+#endif
+
+int main() {
+    stdio_init_all();
+#if !defined(i2c_default) || !defined(PICO_DEFAULT_I2C_SDA_PIN) || !defined(PICO_DEFAULT_I2C_SCL_PIN)
+    #warning i2c/mpu6050_i2c example requires a board with I2C pins
```
Comment (lurch) [inline i2c/pa1010d_i2c/pa1010d_i2c.c:116]: I guess this should have said `i2c/pa1010d_i2c` instead of `i2c/mpu6050_i2c` ?

**Comment by kilograham** [comment]
i'm fine with that if they are effectively the same (modulo actually working!)

**Pull request closed without merging**